include/crocoddyl/multibody/wrench-cone.hxx

```diff
@@ -142,9 +142,12 @@ void WrenchConeTpl<Scalar>::update() {
   Scalar mu = mu_;
   Scalar theta = Scalar(2) * M_PI / static_cast<Scalar>(nf_);
   if (inner_appr_) {
-    mu *= cos(theta / Scalar(2.));
+    mu *= cos(theta * Scalar(0.5));
   }
 
+  // Create a temporary object for computation
+  Matrix3s R_transpose = R_.transpose();
```

Comment:
Should we allocate this data once (i.e., as a member of the structure)? Not sure if it is justified in this case. Any thought?